packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -181,11 +181,15 @@ export class HostedPluginProcess implements ServerPluginRunner {
             return;
         }
         this.logger.error(`[${serverName}: ${pid}] IPC exited, with signal: ${signal}, and exit code: ${code}`);
-        this.messageService.error(
-            'Plugin runtime crashed unexpectedly, all plugins are not working, please reload the page. ' +
-            'It might happen if there is not enough memory for the plugins.',
-            { timeout: 15 * 60 * 1000 }
-        );
+
+        const message = 'Plugin runtime crashed unexpectedly, all plugins are not working, please reload the page.';
+        let hintMessage: string = 'If it doesn\'t help, please check Theia server logs.';
```

Comment:
👍 I don't actually know when a page reload can help. Most likely a user will just reload infinitely.

Reply:
Yeah this was the case when I was getting the error message, I just kept reloading thinking something should happen when I refreshed but it would still continuously error